lib/screens/profile_screens/security/interactor/viewmodels/security_bloc.dart

```diff
@@ -20,7 +21,7 @@ class SecurityBloc extends Bloc<SecurityEvent, SecurityState> {
 
   SecurityBloc({required AuthenticationBloc authenticationBloc})
       : _authenticationBloc = authenticationBloc,
-        super(SecurityState.initial()) {
+        super(SecurityState.initial(remoteConfigurations.featureFlagExportRecoveryPhraseEnabled)) {
```

Comment:
the state should have no knowledge of `remoteConfigurations` we pass the flag value into the state.